core/ibft.go

```diff
@@ -133,18 +134,26 @@ func NewIBFT(
 	}
 }
 
+// SetTime function set duration to gauge
+func SetTime(prefix string, startTime time.Time) {
```

Comment:
Maybe `MeasureTime`? `SetTime` is a bit misleading.

Reply:
`SetMeasureTime` ? :)

Reply:
`MeasureSetTime` ?

Reply:
`SetMeasurementTime` seems as the most professional :joy:

Reply:
fixed: https://github.com/0xPolygon/go-ibft/pull/85/commits/f72719c20174d02291510a82b1742d7719632bb7

SetMeasurementTime is used.